app/validators/questionnaire_validator.py

```diff
@@ -87,10 +87,11 @@ def validate_smart_quotes(self):
                 values_to_check = schema_text.values()
 
             for schema_text in values_to_check:
-                found = quote_regex.search(schema_text)
-
-                if found:
-                    self.add_error(
-                        error_messages.DUMB_QUOTES_FOUND,
-                        pointer=translatable_item.pointer,
-                    )
+                if schema_text is not None:
```

Comment:
I think this might highlight an underlying issue with a `None` value being in `translatable_items`. I think we should test this with translations to see if it breaks it - it could error trying to lookup a translation value of `None`.

Reply:
I tested translation templates and it looks like `None` labels are not being added to pot files, so it looks like it doesn't break translations.

Reply:
Did you test generating a translated schema as well as extraction?

Reply:
I did both and it works fine, `null` labels are not added to .pot files and generating schemas from .pot files is working as expected, I created automated test for eq-translations, should I add it?